docs/source/managed-federation/deployment.mdx

```diff
@@ -344,3 +348,25 @@ On the other side of the equation sits the router. The router can regularly poll
 4. The router continues to resolve in-flight requests with the previous configuration, while using the updated configuration for all new requests.
 
 Alternatively, instead of getting its configuration from Apollo Uplink, the router can specify a path to a supergraph schema upon its deployment. This static configuration is useful when you want the router to use a schema different than the latest validated schema from Uplink, or when you don't have connectivity to Apollo Uplink. For an example of this workflow, see an [example of configuring the router for blue-green deployment](#example-blue-green-deployment).
+
+## Rolling back a deployment
+
+When a deployment has an issue, you may need to roll it back to update its subgraph and supergraph schemas and redeploy it.
+
+Guidelines for rollback:
+
+- For typical managed federation, where Apollo Uplink provides the supergraph schema to the router, we recommend rolling forward for rollback: create a new commit that reverts schema changes, then follow the standard [schema delivery pipeline](/graphos/delivery) to recreate the previous deployment in a new one.   
+
+- For blue-green deployment scenarios, where each deployment typically has its own Docker container image, you can roll back by redeploying the previous image, because its router has the correct supergraph schema embedded in the image itself.
+
+- For small subgraph-only schema changes not requiring resolvers, it may be fastest and effective to rollback only the supergraph schema. This may be applicable for field migrations, such as when using progressive `@override`.
+
+- In general, a rollback for subgraph schema changes likely need to revert both subgraph and supergraph schemas. Such a rollback relies on versioning deployed routers and subgraphs together, and implementing it means first updating and publishing subgraph schemas, then launching and generating a supergraph schema.
+
+- With Kubernetes, you can roll back a router deployment to its previous revision with the following command:
```

Comment:
Would remove this section as we do not recommend rolling back only the supergraph schema on the router in isolation as subgraph compatibility must be taken into account, subsequent subgraph publishes will generate a new supergraph schema that may lose the rolled back changes, and in general it's better to fix the problem in the source of truth which is in the subgraph repository.

If we wanted to keep this example, would be more comfortable having it be an immutable `Deployment` of both **router & subgraphs**, such that the `kubectl rollback` would ensure the router's supergraph schema is compatible with the underlying subgraphs deployed in the target environment.